src/components/TextInput/BaseTextInput.js

```diff
@@ -219,10 +223,29 @@ class BaseTextInput extends Component {
                         </View>
                     </TouchableWithoutFeedback>
                 </View>
-                {!_.isEmpty(this.props.errorText) && (
-                    <InlineErrorText>
-                        {this.props.errorText}
-                    </InlineErrorText>
+                {(!_.isEmpty(inputHelpText) || !_.isNull(this.props.maxLength)) && (
+                    <View
+                        style={[
+                            styles.mt1,
+                            styles.flexRow,
+                            styles.justifyContentBetween,
+                            styles.ph3,
+                        ]}
+                    >
+                        {!_.isEmpty(inputHelpText) && (
+                            <Text style={[formHelpStyles]}>{inputHelpText}</Text>
+                        )}
+                        {!_.isNull(this.props.maxLength) && (
+                            <>
+                                <View style={styles.flex1} />
+                                <Text style={[formHelpStyles]}>
```

Comment:
Not a bit fan of the `View` workaround. Would something like this work?
```suggestion
                                <Text style={[formHelpStyles, styles.flex1, styles.textAlignRight]}>
```